program/res/messager/.binpath

```diff
@@ -0,0 +1 @@
+../../target/wasm32-unknown-unknown/debug/messager
```

Comment:
No need to push

Reply:
fixed (force-pushed to not blow-up history)